c7n/resources/ec2.py

```diff
@@ -77,7 +77,7 @@ class resource_type(object):
     action_registry = actions
 
     # if we have to do a fallback scenario where tags don't come in describe
-    permissions = ('ec2:DescribeTags',)
+    permissions = ('ec2:DescribeTags', 'ec2:DescribeInstanceAttribute',)
```

Comment:
hmm actually this permission should be on the disable filter not on the resource manager.

Reply:
Oops, sorry about that. I was originally taking the approach of adding this code to DescribeEC2.augment() until I realized that DescribeInstanceAttribute needs to be called once per instance, and that would probably be a _really_ bad idea to do for every instance always.